docs/api/qiskit/release-notes/1.1.mdx

```diff
@@ -910,7 +910,11 @@ The Qiskit 1.1.0 release is a minor feature release that includes a myriad of ne
 
 ### Other Notes
 
-*   Support for the arm64 macOS platform has been promoted from Tier 3 to Tier 1. Previously the platform was at Tier 3 because there was no available CI environment for testing Qiskit on the platform. Now that Github has made an arm64 macOS environment available to open source projects [\[1\]](#id9) we’re testing the platform along with the other Tier 1 supported platforms.
+*   Support for the arm64 macOS platform has been promoted from Tier 3 to Tier 1. Previously the platform was at Tier 3 because there was no available CI environment for testing Qiskit on the platform. Now that Github has made an arm64 macOS environment available to open source projects <span id="id8" />[\[1\]](#id9) we’re testing the platform along with the other Tier 1 supported platforms.
+
+    ## Footnotes
```

Comment:
Hm, @frankharkins this wasn't expected in a release note. What do you think? Should we skip adding the h2 if it's a release note file?